v5analytics/visallo
# Cloud Ingest (S3) and Structured File Import (csv, excel, parquet)

- [ ] @kunklejr @sfeng88 @diegogrz
- [ ] @mwizeman

Testing Instructions:
Cloud Ingest: File import -> Cloud Ingest -> S3. Connect with basic/session auth ingest one or more files. When finished, Open should select them.
Structured File: Test with csv, excel, parquet. Example parquet file in web/plugins/structured-ingest/parquet/src/test/resources

CHANGELOG
Added: Cloud Ingest support with new plugin for S3.
Added: Structured file ingest mapping interface. Ingest spreadsheets (csv, excel) and parquet files.


![screens](https://cloud.githubusercontent.com/assets/7098/21571052/1fee1dbe-ce91-11e6-82e5-63562c845e40.png)
